tfhe/src/core_crypto/algorithms/lwe_encryption.rs

```diff
@@ -135,12 +135,12 @@ pub fn encrypt_lwe_ciphertext<Scalar, KeyCont, OutputCont, Gen>(
         lwe_secret_key.lwe_dimension()
     );
 
-    let (mut mask, body) = output.get_mut_mask_and_body();
+    let (mut mask, mut body) = output.get_mut_mask_and_body();
 
     fill_lwe_mask_and_body_for_encryption(
         lwe_secret_key,
         &mut mask,
-        body,
+        &mut body,
```

Comment:
Is this not creating a `&mut` reference that the compiler automatically deref ? 🤔

Reply:
I think this is like doing an &mut &mut with the LweBodyRefMut (could be wrong, I need to read this PR)

Reply:
Ownership of `LweBodyRefMut` is never used in `fill_lwe_mask_and_body_for_encryption` hence the warning
But I agree adding a `&mut` is not really useful

Reply:
The mask already has a `&mut` so I think we should have both or none